components/org.wso2.carbon.privacy.forgetme.apim.datasource/pom.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.wso2.carbon.privacy</groupId>
+        <artifactId>forget-me</artifactId>
+        <version>1.1.1-SNAPSHOT</version>
+        <relativePath>../../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>org.wso2.carbon.privacy.forgetme.apim.datasource</artifactId>
+    <name>[GDPR] Forget Me - APIM Datasources Extension</name>
```

Comment:
Lets generify this component so that any other can later use this component if they want SELECT -> modify -> UPDATE pattern.